TanukiKit/Project.swift

```diff
@@ -174,9 +239,75 @@ public enum VisibilityLevel: Int {
     }
 }
 
+@objc open class EventData: NSObject {
+    open var objectKind: String?
+    open var eventName: String?
+    open var before: String?
+    open var after: String?
+    open var ref: String?
+    open var checkoutSha: String?
+    open var message: String?
+    open var userID: Int?
+    open var userName: String?
+    open var userEmail: String?
+    open var userAvatar: URL?
+    open var projectID: Int?
+    open var project: Project?
+    open var commits: [Commit]?
+    open var totalCommitsCount: Int?
+    //open var repository: Repository? // TODO: Repo Requests
+
+    public init(_ json: [String: AnyObject]) {
+        objectKind = json["object_kind"] as? String
+        eventName = json["event_name"] as? String
+        before = json["before"] as? String
+        after = json["after"] as? String
+        ref = json["ref"] as? String
+        checkoutSha = json["checkout_sha"] as? String
+        message = json["message"] as? String
+        userID = json["user_id"] as? Int
+        userName = json["user_name"] as? String
+        userEmail = json["user_email"] as? String
+        if let urlString = json["user_avater"] as? String, let urlFromString = URL(string: urlString) {
+            userAvatar = urlFromString
+        }
+        projectID = json["project_id"] as? Int
+        project = Project(json["project"] as? [String: AnyObject] ?? [:])
+        commits =  (json["commits"] as? [[String: AnyObject]]).map { Commit($0) } // TODO: Check class list parsing
```

Comment:
Well, on compilation it gives me: `Cannot convert value of type '[[String : AnyObject]]' to expected argument type '[String : Any]'`
But if I use only one pair of brackets it tells me: `Cannot convert value of type 'Commit' to closure result type '[Commit]'`

BTW: Happy New Year! 🎉